### PR TITLE
Add Community to website navbar, add events to community as subfolder

### PR DESCRIPTION
**Pull request by mrcdb**
This PR addresses open issues mentioned in https://github.com/cncf/tag-security/issues/1257 regarding the **Events** content that is currently conveyed on the TAG security website.

In particular, the PR:
- enables the top-level `community` folder as a navbar item in the website in place of the existing events page, which is currently managed from  `website/content`
- creates a `community/events` subfolder in the repository
- moves existing markdown files related to events to the `community/events subfolder` (also updating naming convention so they all show with hyphens on the website)
- creates README.md files for `community/` and  `community/events` for improved website navigation experience
- removes boilerplate community HTML layout

### CODE REVIEW

**Comment by mrcdb** [comment]
I have been trying to fix the linter links check but it looks like a couple of sched links keep returning 429 (too many requests), and I suspect its due to the large amount of links being tested in this PR. I can open the links fine from my laptop. Is there a workaround for that?

**Review by brandtkeller — COMMENTED** [review]
Thanks for grabbing this off of the #1257 TODO list! Each iteration helps with finding a state of flow. 

Community as a navbar item feels appropriate in my opinion. The placement of events among a list of initiatives feels like something we can refine. (maybe we can nest the initiatives under a sub-directory?). 

Thoughts on a structure such as:

```bash
.
|-- community
|   |-- events
|   |   |-- README.md
|   |-- working-groups
|   |   |-- README.md
|   |   |-- controls
|   |   |   |-- README.md
|   |-- research
|   |   |-- README.md
```

CC: @eddie-knight @anvega

File: website/themes/docsy/layouts/community/list.html
Comment (eddie-knight) [inline website/themes/docsy/layouts/community/list.html:1]: @mrcdb could you summarize the value of removing this theme file?

File: community/events/safe-kubecon.md
Comment (eddie-knight) [inline community/events/safe-kubecon.md:1]: This highlights an issue regarding sorting past events and future events.

Having this file within events is a dramatic improvement from having it at the repo top level, but does it now require website visitors to click and read the entry to determine whether it is a past or upcoming event? If so, do we have any way to improve that?

Happy to file this as a nit to address in a followup PR, if we have some type of consensus the matter.

File: website/themes/docsy/layouts/community/list.html
Comment (eddie-knight) [inline website/themes/docsy/layouts/community/list.html:1]: @mrcdb @brandtkeller 

What if we can set this up to behave like the Blog page? Might be as simple as overriding this layout file with a copy of the blog template. Might also take more work, but WDYT— worth a try?

**Comment by eddie-knight** [comment]
> Community as a navbar item feels appropriate in my opinion. The placement of events among a list of initiatives feels like something we can refine. (maybe we can nest the initiatives under a sub-directory?).

I like the idea. Stirs up more ideas about community structure... but I'll save those for later.

**Comment by mrcdb** [comment]
> Community as a navbar item feels appropriate in my opinion. The placement of events among a list of initiatives feels like something we can refine. (maybe we can nest the initiatives under a sub-directory?).

@brandtkeller I thought the same when testing out the changes. Having the events page at the same level as each working group sounds a bit inconsistent, I'd second your idea of having a "working groups" folder that includes all the WGs collaterals as sub-folders. Happy to give a try at this in this PR.

**Comment by eddie-knight** [comment]
> @brandtkeller I thought the same when testing out the changes. Having the events page at the same level as each working group sounds a bit inconsistent, I'd second your idea of having a "working groups" folder that includes all the WGs collaterals as sub-folders. Happy to give a try at this in this PR.

What if we modify the directories to have WG at the end of each? We could open this up for discussion as a followup item.

File: website/themes/docsy/layouts/community/list.html
Comment (mrcdb) [reply website/themes/docsy/layouts/community/list.html:1]: Blog posts get grouped by year based on their heading, e.g.:

```markdown
---
title:  "Clusters for all cloud tenants"
date:   2022-06-02 13:04:00 +0200
author: Josh Gavant 
---
```

I think the best approach would be to create standalone files for each event (both past and future, e.g. each CNSCon event gets its own page) and creating a heading for each of them so that they are automatically grouped based on the year. This would improve readability and fix the sorting issue you mentioned above, but it requires a bit more work as individual files are created for each event.

Comment (eddie-knight) [reply website/themes/docsy/layouts/community/list.html:1]: That matches the approach I used for the image above. If we add the headers to the event files, they will behave like blog posts within this subsection. Then we'd be able to remove the past and future events files.

Comment (eddie-knight) [reply website/themes/docsy/layouts/community/list.html:1]: @brandtkeller @mrcdb Do yall want me to push the commit to this PR or in a follow-up?

Comment (mrcdb) [reply website/themes/docsy/layouts/community/list.html:1]: I can help with updating the event files and checking in the blog-alike layout for community events myself, but please feel free to push any commit to this PR if you have already built some of that :+1:

Comment (mrcdb) [reply website/themes/docsy/layouts/community/list.html:1]: @eddie-knight I have pushed an initial commit that would add the events on a timeline similarly to what is done for blogs. the only problem I couldn't fix is how to apply the custom layout to `community/events`, not sure if this is a Hugo limitation (I am not really an expert on that).

As a workaround, I had to move Events as a top-level menu item so that the custom layout would only apply to the events and not affect the other pages in `community`, which otherwise wouldn't display correctly.

**Comment by anvega** [comment]
Dottore @mrcdb! Nice one.

To respond to @brandtkeller, I don't have strong thoughts on a final structure. It's all fluid, and we can continue to iterate until we arrive at what it should be for now, although the thinking is synchronized with #1283. As part of the lift there, I created an events directory and moved two of the three files. I removed the `safe-kubecon.md` as it was notes that, due to lack of context, don't mean much to new visitors. It probably still deserves an entry under past events but without the hyperlink to the markdown file.

If you have a look at this [document](https://docs.google.com/document/d/1bdcZ3WIBmLrTTZj_1aLp3fGl68WqEhifItIHThE53k8/edit#heading=h.3ckk7h76z9jg), you can get a view of the current target structure of the repo org. I deliberately didn't contemplate website impact there, so I'll defer on that to you all. It hadn't really been part of the workflow on maintaining the repo until your work that is underway here.

As part of that first #1283, I also moved the workgroup subdirectories under a `working-groups` directory. There are a few more changes like moving assessments to `community/resources` and moving the `Makefile` and `dockercompose.yml` inside the CI folder and adjusting the Actions workflow, which I'm still working on. I should probably create individual issues and a milestone tracker for visibility for the rest of the work and for us to be in sync.

I didn't create markdown READMEs for `community` or its subdirectories, so this is a super welcome addition to fix that.

**Comment by anvega** [comment]
From a sequencing standpoint, we should probably review #1273 given its size, as merging this one first will result in conflicts that won't be easily resolved.

File: website/themes/docsy/layouts/community/list.html
Comment (eddie-knight) [reply website/themes/docsy/layouts/community/list.html:1]: The last commit seems to be a large divergence from your previous approach 🤔 If you'd like to roll it back, I can make a small commit that allows for a similar result while keeping the Events content in the Community directory.

**Comment by eddie-knight** [comment]
> From a sequencing standpoint, we should probably review #1273 given its size, as merging this one first will result in conflicts that won't be easily resolved.

@anvega, The other PR is currently failing, but happy to review it once it's out of WIP. It's a massive change set, so we'll need at least two thorough reviews on it.

@mrcdb I left a comment above— I don't think this PR is in a good state right now, but if we roll it back I'm happy to add a follow-up PR or commit with the changes needed to get the bloggy behavior in `community/events`

File: website/themes/docsy/layouts/community/list.html
Comment (mrcdb) [reply website/themes/docsy/layouts/community/list.html:1]: Yep, I did try to make it work (see my comment above) but faced some limitations. I have reverted my previous commit, please feel free to push your commit :+1:

**Comment by mrcdb** [comment]
> @mrcdb I left a comment above— I don't think this PR is in a good state right now, but if we roll it back I'm happy to add a follow-up PR or commit with the changes needed to get the bloggy behavior in `community/events`

Agree on that, I just rolled this back to the previous state. Happy for you to add a commit in the repo to get the desired behaviour for `community/events` (as I wasn't able to get that working without affecting other community pages unless I moved it to a top-level menu item)

**Comment by eddie-knight** [comment]
Good shout— merging!

**Review by eddie-knight — APPROVED** [review]
LGTM! Thanks Marco.
